src/webui/components/notification/elementary/RatedCommentReaction.tsx

```diff
@@ -0,0 +1,18 @@
+
+import { AggregatedData as ReactionView } from '^/domain/reaction/aggregate/types';
+import { Border, Column, Text } from '^/webui/designsystem';
+
+type Props = {
+    readonly reaction: ReactionView;
```

Comment:
```suggestion
    readonly comment: string;
```

Same reasoning as the `RatedComicReaction` component.